### PR TITLE
Brightness API

### PR DESCRIPTION
**Pull request by malverick**
Tested and working fine.
**Tested on**
**Device** - Xiaomi Redmi Note 4
**Android API** - 19
**Android SDK** - 20
**Android NDK** - 9c
whatsapp image 2017-06-30 at 1 19 08 pm

### CODE REVIEW

**Comment by malverick** [comment]
I have improved the UI of example, now you don't have to press `Set Brightness` button to change brightness. As the slider value changes, brightness changes automatically.
fullsizeoutput_81

File: examples/brightness/main.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<BrightnessInterface>:
+    orientation: 'vertical'
+    Label:
+        text: 'Adjust the slider to increase \\n or decrease the brightness'
+    Slider:
+        id: slider
+        min: 0
+        max: 100
+        value: root.get_current_brightness()
+        on_touch_up: root.set_brightness(slider.value)
```
Comment (akshayaurora) [inline examples/brightness/main.py]: you might want to try using on_value to get brightness to update while the slider is moving.

**Comment by akshayaurora** [comment]
@malverick thanks nice work, looking forward for Linux, Windows, OSX implementations too.

**Comment by malverick** [comment]
Your welcome @akshayaurora . I have started working on linux part.

**Pull request merged**